app.js

```diff
@@ -1,10 +1,15 @@
 const Srf = require('drachtio-srf');
-const srf = new Srf();const config = require('config');
+const srf = new Srf();
+const config = require('config');
 const logger = require('pino')(config.get('logging'));
-const regParser = require('drachtio-mw-registration-parser') ;
```

Comment:
It seems like it might be worth leaving regParser middleware in, because it does some things you do below in the code  -- parsing aor, figuring out Expires etc.